Upstream-Status: Inappropriate [embedded specific]

help2man is looking at the generated binary for help output. This does not work for cross compilations. So taking out the local PREPATH (../src) directory from path so that help2 man can find the native version of the bison in the native sysroot directory.

Date: 2010/06/28
Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>

diff --git a/doc/Makefile.am b/doc/Makefile.am
index c7f2755..af9453a 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -80,7 +80,6 @@ PREPATH = $(top_builddir)/src
 	  echo cd $$dir '&&' $(MAKE) $(AM_MAKEFLAGS) $$program &&	\
 	  (cd $$dir && $(MAKE) $(AM_MAKEFLAGS) $$program) || exit;	\
 	done
-	$(AM_V_at)PATH="$(PREPATH)$(PATH_SEPARATOR)$$PATH";		     \
 	export PATH;							     \
 	$(HELP2MAN)							     \
 	    --include=$*.x						     \
